src/inc-lib-variants.php

```diff
@@ -194,7 +195,7 @@ function lovd_fixHGVS ($sVariant, $sType = 'g')
                 return lovd_fixHGVS(
                     $sBeforeSuffix . $aVariantInfo['type'] . str_replace(array('(', ')'), '', $sSuffix), $sType);
 
-            } elseif (preg_match('/^\[[^NX][^;]+\]$/', $sSuffix)) {
+            } elseif (preg_match('/^\[[^NX][^;]*]$/', $sSuffix)) {

```

Comment:
Why was this changed back? Did I not understand what kind of possible values are matched here?

By the way, `]` does not strictly need an `\` in front of it, but I find it easily confused with the `]` which means the end of a range of characters that can be expected. So I prefer escaping `]` the same way `[` needs to be escaped.